Documentation/README_BoundsControl.md

```diff
@@ -40,6 +40,9 @@ Adds a padding to the collider bounds used to calculate the extents of the contr
 Indicates whether the control is flattened in one of the axes, making it 2 dimensional and disallowing manipulation along that axis. This feature can be used for thin objects like slates.
 If flatten axis is set to *Flatten Auto* the script will automatically pick the axis with the smallest extent as flatten axis.
 
+### Uniform Scale On Flattened Axis
+Indicates whether to scale along the flattened axis when uniform scale is used. If set to false, uniform scaling will not happen along the flattened axis. Note when using non-uniform scaling, the flattened axis is never scaled along.
```

Comment:
Might be worth mentioning that the default value for this property is true which means, if a specific axis is selected to flatten, you also need to set the `UniformScaleOnFlattenedAxis` property to false for the flatten axis behavior to register.  

For example:
There could be potential confusion if someone selects `FlattenXAxis`, then when they test it out they see that there is still uniform scaling along all axes.  Scaling along the X axis persists until they set `UniformScaleOnFlattenAxis` to false.  Although I do understand the need to maintain existing behavior.